programs/dynamic-bonding-curve/src/instructions/partner/ix_create_config.rs

```diff
@@ -224,10 +243,22 @@ impl ConfigParameters {
         self.locked_vesting.validate()?;
 
         // validate migrate fee option
-        require!(
-            MigrationFeeOption::try_from(self.migration_fee_option).is_ok(),
-            PoolError::InvalidMigrationFeeOption
-        );
+        let migration_fee_option = MigrationFeeOption::try_from(self.migration_fee_option)
+            .map_err(|_| PoolError::InvalidMigrationFeeOption)?;
+
+        if migration_fee_option == MigrationFeeOption::Customizable {
+            require!(
+                self.migrated_pool_fee.is_some(),
+                PoolError::InvalidMigratedPoolFee
```

Comment:
need to check if migration_fee_option != MigrationFeeOption::Customizable, then self.migrated_pool_fee.is_none()

Reply:
let migrated_pool_fee = self.migrated_pool_fee.ok_or_else(||PoolError::InvalidMigratedPoolFee)?;